test/cql-pytest/conftest.py

```diff
@@ -69,5 +69,16 @@ def test_keyspace(cql):
     yield name
     cql.execute("DROP KEYSPACE " + name)
 
+# The "scylla_only" fixture can be used by tests for Scylla-only features,
```

Comment:
i'm not too familiar with this pytest implementation, but doesn't it have a logging of the tested version at all? meaning, like Dtest has a scylla_version parameter or a log print for that.

Reply:
The idea of cql-pytest is that it uses **CQL** (i.e., the Python Cassandra driver) to communicate with the server - that can be either Scylla of any version, Cassandra, one node or a cluster of 20 nodes, etc. When we run Scylla (`test/cql-pytest/run` does this automatically), Scylla's log contains whatever it usually prints out (including the version), and this is available in Jenkins for example; But the test itself does not have access to this log - just to CQL requests. That's intentional.